### DISCUSSION TITLE
How to map K to show docstrings in pylsp?

### DISCUSSION THREAD

**Discussion by ubaldot** [yegappan/lsp#560] [Q&A] [answered]
Pressure of `K` trigger the command `:!python3 - pydoc 'text_under_the_cursor'` and I am wondering if it is possible to change such a behavior in such a way that when I hit `K` a popup/balloon showing the docstring of the word under the cursor is shown.

**Comment by yegappan** [comment, marked as answer]
As described in the help for the `:LspHover` command, you can use the following setting:

```
:set keywordprg=:LspHover
```

With this, if you press `K` on a keyword, it will invoke the `:LspHover` command.
